frontend/test/metabase/scenarios/joins/joins.cy.spec.js

```diff
@@ -293,7 +293,8 @@ describe("scenarios > question > joined questions", () => {
       cy.findByText("12928_Q2");
     });
 
-    cy.findAllByText(/Products? → Category/).should("have.length", 2);
+    cy.findAllByText(/Products? → Category/).should("have.length", 1);
+    cy.findAllByText(/Question \d+? → Category/).should("have.length", 1);
```

Comment:
Note the column should be `/Question \d+? → Category/` and not `12928_Q2 -> Category` as you might expect. We currently use "Question <id> -> <column name>" format for the display name of columns from nested queries. See https://github.com/metabase/metabase/issues/21646